### DISCUSSION TITLE
Работа с транзакциями

### DISCUSSION THREAD

**Discussion by mxv2** [ardov/zerro#63] [Ideas]
На сколько понял, я могу только исправить транзакцию, но не завести новую.
API Zenmoney вроде бы позволяет такое сделать.

Планируется ли такая фича? Можно ли законтрибьюить и тогда, какие правила на контрибьют?

**Comment by ardov** [comment]
Привет, да, сейчас можно только исправить существующую транзакцию.
Создание хочется в будущем добавить, но пока соотношение польза/сложность не в его пользу) Создание операций хорошо сделано в ДМ, поэтому для меня пока работа категориями приоритетней. А для каких случаев создание операций в зерро кажется полезным?

По поводу самого создания операций, там есть много нюансов, что бы это выглядело и работало хорошо. Например отдельная тема – работа с мерчантами, там надо разобраться в каких случаях стоит создавать нового мерчанта, а в каких достаточно просто указать поле payee. Но это, конечно, уже не MVP) Вообще, я думал в будущем создание просто вырастить из формы редактирования, это кажется логичным.

Я бы сейчас не контрибьютил, т.к. там у меня пока висит ветка с переездом на MUI 5 и кучей изменений) Плюс, думаю, в ней же появятся полезные компоненты для создания категорий типа селекта счетов.

**Reply by mxv2** [reply]
Спасибо за ответ.
После YNAB ввод транзакций в ZenMoney как-то совсем не смотрится :)

"А для каких случаев создание операций в зерро кажется полезным?" - мне лично было бы удобно пользоваться одним клиентом. На счет полезно не уверен.

**Reply by vrslev** [reply]
Zerro по определению не может полностью заменить официальное приложение. Для меня киллер-фича Дзен-мани в целом — синхронизация с банками. Очевидно, Zerro не сможет имплементировать что-то подобное — и это ок. Идеальный, реалистичный workflow такой: счета в Дзен-мани постоянно синхронизируются; раз в день или неделю просматриваю операции в официальном приложении, после этого сразу иду в Zerro за статистикой и инсайтами на следующий период времени.

**Reply by mxv2** [reply]
Так Zerro уже умеет редактировать транзакцию. Это почти тоже самое, что создать новую. В этом и был вопрос.
При чем тут синхронизация?

**Reply by vrslev** [reply]
Я больше про Zerro как полноценная замена официальному клиенту)
